navigation/CameraFlowNavigator.tsx

```diff
@@ -0,0 +1,65 @@
+import { useNavigation } from "@react-navigation/native";
+import React, { FunctionComponent, ReactElement, useState } from "react";
+import { View } from "../components/Themed";
+import CameraRecording from "../screens/CameraRecording";
+import LetsStartRecording from "../screens/LetsStartRecording";
+import PrepareFace from "../screens/PrepareFace";
+import { Text } from "../components/Themed";
+
+interface CameraFlowNavigatorProps<T> {
+  objects: T[];
+  overlayCreator: (object: T) => ReactElement;
+}
+
+enum State {
+  prepareFace,
+  letsStart,
+  recording,
+}
+
+function CameraFlowNavigator<T>({
+  objects,
+  overlayCreator,
+}: CameraFlowNavigatorProps<T>): ReactElement {
+  // Right now we are using the default values, since we are not sure the best way to pass in the props
+  // In a follow-up pr we will deal more with the navigation aspect
+  let objectValues = objects || [1, 2, 3];
```

Comment:
Oh, is it supposed to take in the Episodes? And then these are the numbers?

Reply:
Nevermind, I think I get it (in the future, this will be passed something, and we won't be using the plain numbers."

Reply:
Yep you got it!